### PR TITLE
Add links to the C# 101 video series

### LINKED ISSUE #15008: The C# tutorials page should link to the new C# 101 videos
@shanselman and @kendrahavens created a series of [great C# 101 videos](https://www.youtube.com/playlist?list=PLdo4fOcmZ0oVxKLQCHpiUWun7vlJJvUiN)

They should be linked from the following pages:

- [x] C# Guide landing page
- [x] C# Guide tutorials page
- [x] C# Guide Beginner tutorials home page.

/cc @JaySing

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #15008

Provide links to the C# 101 video series from the C# home page, the tutorials home page, and the intro to C# tutorials home page.

Internal review links:
[C# Guide index page](https://review.docs.microsoft.com/en-us/dotnet/csharp/index?branch=pr-en-us-15605)
[C# Tutorials index page](https://review.docs.microsoft.com/en-us/dotnet/csharp/tutorials/index?branch=pr-en-us-15605)
[Intro to C# tutorial index page](https://review.docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/index?branch=pr-en-us-15605)

/cc @JaySing @shanselman @kendrahavens

### CODE REVIEW

**Comment by BillWagner** [comment]
I'm trying one page with the first video in the series embedded in the page. Comments on that experience?

File: docs/csharp/tutorials/index.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Welcome to the C# tutorials. These start with interactive lessons that you can r
 
 ## Introduction to C# interactive tutorials
 
+If you want to start your exploration in video format, [this series](https://aka.ms/dotnet3-csharp) provides an introduction to C#. You'll learn about concepts you can explore in this tutorials.
```
Comment (pkulikov) [inline docs/csharp/tutorials/index.md]:
```suggestion
If you want to start your exploration in video format, [this series](https://aka.ms/dotnet3-csharp) provides an introduction to C#. You'll learn about concepts you can explore in these tutorials.
```

**Comment by BillWagner** [comment]
I like the embedded video on the Intro to C# tutorial page. Other opinions?

**Comment by gewarren** [comment]
@ghogen Did we ever add links to the videos from the VS docs?

File: docs/csharp/tutorials/intro-to-csharp/index.md [hunk 2]
```diff
@@ -7,7 +7,10 @@ ms.custom: mvc
 # Introduction to C\#
 
 Welcome to the introduction to C# tutorials. These start with interactive lessons
-that you can run in your browser.
+that you can run in your browser. You can learn the basics of C# from
+[this video series](https://aka.ms/dotnet3-csharp) before starting these interactive lessons.
```
Comment (Youssef1313) [inline docs/csharp/tutorials/intro-to-csharp/index.md]:
```suggestion
[C# 101 video series](https://aka.ms/dotnet3-csharp) before starting these interactive lessons.
```

File: docs/csharp/index.md [hunk 3]
```diff
@@ -14,6 +14,8 @@ helpviewer_keywords:
 The C# guide provides many resources about the C# language. This site has many different audiences. Depending on your experience with programming, or with the C# language and .NET, you may wish to explore different sections of this guide.
 
 - For brand-new developers:
+  - Watch this [video series](https://aka.ms/dotnet3-csharp). It provides an introduction to the important concepts you'll explore in these tutorials.
```
Comment (Youssef1313) [inline docs/csharp/index.md]:
```suggestion
  - Watch [C# 101 video series](https://aka.ms/dotnet3-csharp). It provides an introduction to the important concepts you'll explore in these tutorials.
```

File: docs/csharp/tutorials/index.md [hunk 4]
```diff
@@ -9,6 +9,8 @@ Welcome to the C# tutorials. These start with interactive lessons that you can r
 
 ## Introduction to C# interactive tutorials
 
+If you want to start your exploration in video format, [this series](https://aka.ms/dotnet3-csharp) provides an introduction to C#. You'll learn about concepts you can explore in these tutorials.
```
Comment (Youssef1313) [inline docs/csharp/tutorials/index.md]:
```suggestion
If you want to start your exploration in video format, [C# 101 video series](https://aka.ms/dotnet3-csharp) provides an introduction to C#. You'll learn about concepts you can explore in these tutorials.
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
I think this is a better link text.

**Review by Thraka — APPROVED** [review]

**Comment by mairaw** [comment]
I like it @BillWagner!

**Pull request merged**